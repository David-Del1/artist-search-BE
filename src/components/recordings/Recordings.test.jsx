import React from 'react';
import { render } from '@testing-library/react';

describe('recordings Component', () => {
  it('li with h2 inside', () => {

    const { asFragment } = render (
      <li>
        <h2>Hi I&apos;m guessing</h2>
      </li>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
