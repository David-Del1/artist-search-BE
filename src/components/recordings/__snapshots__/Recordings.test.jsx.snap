// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`recordings Component li with h2 inside 1`] = `
<DocumentFragment>
  <li>
    <h2>
      Hi I'm guessing
    </h2>
  </li>
</DocumentFragment>
`;
